Bug/sql injection vul scan 3

jaysonmc opened the pull request:
Resolves #389

Adding migration to change column type to string in the DB (which GraphQL was already assuming was the case).

Everyone will be required to
1. Rebuild postgres container (to have the init.sh script run to add in uuid support)
2. Re-run migrations from scratch after this change. Previous migrations were modified.

jaysonmc commented:
Also, the internet tells me where is no schema DSL for the DB::statement('ALTER TABLE users ALTER COLUMN id SET DEFAULT uuid_generate_v4();'); part, which is why it is not used there, and 'raw sql' is used instead

tristan-orourke reviewed (changes requested):
I'm generally happy with this. A couple of minor points:
- I think you can more succinctly create the primary key with `$table->uuid('id')->primary();`
- Any particular reason why you didn't use [this function](https://www.postgresql.org/docs/current/functions-uuid.html), which is accessible without loading a module?

jaysonmc commented:
@tristan-orourke 

Good call on both fronts. Simplified two lines to one, and I didn't use that only because ... I did not know it existed. I have just blindly been using the other one for some time now. So, always happy to find a better way to do things! I removed the new dependency and used the function you recommended instead.

tristan-orourke reviewed (changes requested):
One last thing. I just took another glance at [Eloquent docs](https://laravel.com/docs/8.x/eloquent#primary-keys), they say that models that don't use integer ids should specify that using `protected $keyType = 'string';`. I'm not sure what the consequence is exactly, but we should probably follow suit.

jaysonmc commented:
@tristan-orourke 

Added the requested line and fixed a bug in the migrations
